src/twisted/web/test/test_http.py

```diff
@@ -2543,11 +2541,20 @@ def process(self):
         self.assertEqual(len(processed), 1)
         self.assertEqual(processed[0].args, {b"text": [b"abasdfg"]})
 
+        # Now check the disabled case:
+        self.assertEqual(MyRequest.parseBody, True)
+        MyRequest.parseBody = False
```

Comment:
Explicitly making a class-var style public interface like this is fragile; I'd much rather we elide the class-var-level defaults everywhere if we can. Get the parameter to the constructor when it's required.